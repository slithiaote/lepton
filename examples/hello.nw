\documentclass[paper=a7]{scrartcl}
\usepackage[width=7cm,height=10cm]{geometry}
\usepackage{float} \newfloat{leptonfloat}{H}{lol}
\begin{document}
The code below sends "hello world" instructions to the \verb ocaml  interpreter.
<<hello_world -exec ocaml -chunk ocaml>>=
let msg = "Hello world.";;
print_string(msg); print_newline();;
@ 
\end{document}
