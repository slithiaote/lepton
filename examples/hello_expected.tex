\documentclass[paper=a7]{scrartcl}
\usepackage[width=7cm,height=10cm]{geometry}
\usepackage{float} \newfloat{leptonfloat}{H}{lol}
\begin{document}
The code below sends "hello world" instructions to the \verb ocaml  interpreter.
\begin{leptonfloat}
\caption{hello\_world}
\label{hello_world}\vspace*{-\leptonlb}\footnotesize{\texttt{}}\vspace*{-\leptonlc}
\begin{verbatim}
let msg = "Hello world.";;
print_string(msg); print_newline();;
\end{verbatim}
\vspace*{-\leptonld}Interpret with \texttt{ocaml}\vspace*{-\leptonle}
\begin{verbatim}

val msg : string = "Hello world."
Hello world.
- : unit = ()
\end{verbatim}
\end{leptonfloat}
\end{document}
